### ISSUE TITLE
Deactivating module crashes

### ISSUE THREAD

**Issue by elbotho** [humhub/humhub#2055]
When I try to deactivate any module in any space I get the following error and the module is not deactivated.

``` php
Unknown Method – yii\base\UnknownMethodException

Calling unknown method: humhub\modules\content\components\ContentContainerSettingsManager::deleteAll()
```

tell me if you need the full log, but the relevant line is here:
``` php
2. in …/protected/humhub/modules/content/components/ContentContainerModule.php at line 133 – yii\base\Component::__call('deleteAll', [])`
```

HumHub is at **1.1.1**
All modules are up to date.

**Comment by luke-** [comment]
Should be fixed in the upcoming 1.1.2 release in the next days.
https://github.com/humhub/humhub/commit/037ace60a8285d47cfde78b40e8d92d29ffa3a9b

**Comment by elbotho** [comment]
Thanks! 👍
